front/app/containers/Admin/projects/project/projectHeader/ReviewFlow/index.tsx

```diff
@@ -57,21 +46,22 @@ const ReviewFlow = ({ project }: { project: IProjectData }) => {
   }
 
   const approvalPending = projectReview?.data.attributes.state === 'pending';
-  const approvalGranted = projectReview?.data.attributes.state === 'approved';
-
-  const showProjectApprovalButton = approvalPending && canReview;
-  const showReviewRequestButton =
-    isProjectReviewEnabled && !canReview && !approvalGranted;
```

Comment:
We don't need the !approvalGranted check. If project_review is on and you "can't review" (or "can't publish" if my changes get merged) a project, it automatically means it's not been approved.

Reply:
I'm not a huge fan of conflating the two. It happens to work with the current implementation, but mostly "by chance". I think the two concepts are semantically different enough to deserve their own permission rules. As a moderator, the fact that a project has been approved does not give you the permission to review it.